<launch>

  <arg  name="utra_ns" />
  <param name = "utra_ns" type="str" value = "$(arg utra_ns)"/>
  <node
    name="ut_joint_trajectory"
    pkg="utra_controller"
    type="joint_trajectory_1000"
    respawn="false"
    output="screen"
    >
    </node>
    
</launch>